mne/decoding/tests/test_time_gen.py

```diff
@@ -171,6 +171,25 @@ def test_generalization_across_time():
     assert_true(len(gat.test_times_['slices'][0]) == 15 ==
                 np.shape(gat.scores_)[1])
 
+    # Test score_mode
+    gat.score_mode = 'foo'
+    assert_raises(ValueError, gat.score, epochs)
+    gat.score_mode = 'per-fold'
+    scores = gat.score(epochs)
+    assert_array_equal(np.shape(scores), [15, 15, 5])
+    gat.score_mode = 'no-cv'
```

Comment:
no I meant {insert mode here}-wise, just confirming your latest proposal

On Fri, Apr 29, 2016 at 4:46 PM Jean-Rémi KING <notifications@github.com>
wrote:

> In mne/decoding/tests/test_time_gen.py
> <https://github.com/mne-tools/mne-python/pull/3143#discussion_r61588830>:
>
> > @@ -171,6 +171,25 @@ def test_generalization_across_time():
> >      assert_true(len(gat.test_times_['slices'][0]) == 15 ==
> >                  np.shape(gat.scores_)[1])
> >
> > +    # Test score_mode
> > +    gat.score_mode = 'foo'
> > +    assert_raises(ValueError, gat.score, epochs)
> > +    gat.score_mode = 'per-fold'
> > +    scores = gat.score(epochs)
> > +    assert_array_equal(np.shape(scores), [15, 15, 5])
> > +    gat.score_mode = 'no-cv'
>
> mode-wise seems a good way of calling referring to the modes
>
> ? You mean, mode-wise as opposed to score_mode? We already use
> predict_mode for a related functionality, so score_mode seems more
> consistent.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3143/files/2c0fa419e56dd3ab6a5fe679b573acda56d21bf0#r61588830>
>